manual/library/builtin.etex

```diff
@@ -181,7 +181,9 @@ exception Invalid_argument of string
 \index{Invalidargument@\verb`Invalid_argument`}
 \begin{ocamldocdescription}
    Exception raised by library functions to signal that the given
-   arguments do not make sense.
+   arguments do not make sense.  The string gives some information
+   to the programmer.  As a general rule, this exception should not
+   be caught, the code should be modified instead not to trigger it.
```

Comment:
I would add: 

As a general rule, this exception should not be caught, *it denotes a programming error* and the code...